board/fischertechnik/TXT/rootfs/etc/init.d/S80touchscreen-calibration

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+start() {
+  if [ -f /home/ftc/.reset-touchscreen-calibration ]; then
```

Comment:
This is ugly, because it relies on knowing internals of the backend script (where the flag file is located and how it is named). Either change `reset` to only reset the calibration settings if, or introduce a new 'is-uncommitted` (or something like this) case in `calibrate-touchscreen` like this
```
case is-uncommitted)
      if [ -f /home/ftc/.reset-touchscreen-calibration ]; then
          exit 0;
      else 
           exit 1;
      fi
     ;;
```
and change this line to 
```suggestion
  if calibrate-touchscreen is-uncommitted; then